clientGQL_test.go

```diff
@@ -87,6 +87,14 @@ func NewTestRequest(request string, variables string, response string) TestReque
 	if !IsValidJson(testRequest.Variables) {
 		panic(fmt.Errorf("testRequest Variables is not valid json: '%s'", testRequest.Variables))
 	}
+	jsonRequestWithVariables := fmt.Sprintf(
```

Comment:
hmm - my initial hope with this was that we wouldn't need to do this.  Maybe it requires a change to autopilot first but i'd love to just use 
```
testRequest := TestRequest{
		Request:   Templated(request),
		Variables: Templated(variables),
		Response:  Templated(response),
	}
	```
and not have to do any fo the `fmt.Sprintf(` stuff